src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (num) {
+      return this.result + num;
+    },
+    subtract: function (num) {
+      return this.result - num;
+    },
+    multiply: function (num) {
+      return this.result * num;
+    },
+    divide: function (num) {
+      return this.result / num;
+    },
+    reset: function (num) {
+      this.result = 0;
+
+      return this;
+    },
+    operate: function (callback, num) {
```

Comment:
[CODE KNOWLEDGE] - Use the method shorthand for the 'operate' method as well.

Reply:
Not fixed